### PR TITLE
update: print the error code with meaningful name

### PR DESCRIPTION
**Pull request by NopeDl**
### What problem does this PR solve?

Issue Number: #2185 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?):

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

**Comment by wuhongsong** [comment]
`cicheck`  will trigger ci

**Comment by wuhongsong** [comment]
cicheck

**Comment by Ziy1-Tan** [comment]
Enum instead of const is preferred.

File: src/mds/chunkserverclient/chunkserver_client.cpp [hunk 1]
```diff
@@ -48,6 +48,32 @@ namespace curve {
 namespace mds {
 namespace chunkserverclient {
 
+void ChunkServerClient::printErrorCode(int errorCode){
```
Comment (wu-hanqing) [inline src/mds/chunkserverclient/chunkserver_client.cpp:51]: Hi, you shouldn't print error directly in here.

What we need it a function that can transfer error code to corresponding name like https://github.com/NopeDl/curve/blob/0a8642a53c801fc3f29c4e5ebdd5bb304a1e9301/src/client/client_common.h#L241, and when we encounter an error, we can use that function to print its name.

BTW, there are a number of status codes that need to be modified and adjusted, and you can start by doing a general search and listing them.

**Pull request closed without merging**